contracts/WildcardSteward_v2.sol

```diff
@@ -103,17 +102,42 @@ contract WildcardSteward_v2 is Initializable {
         _;
     }
 
-    function initialize(
-        address _assetToken,
-        address _admin,
-        uint256 _patronageDenominator
-    ) public initializer {
+    function initialize(address _assetToken, address _admin)
+        public
+        initializer
+    {
         assetToken = ERC721Patronage_v1(_assetToken);
```

Comment:
For posterity, this initializer trusts that what was passed in was the real one. trivial to verify after deployment, but one of the risks of having it not perform the deployment itself.  (although it definitely balloons the deployment size, so this decision to separate the contract size might be reasonable/required, depending on contract complexity)

Reply:
Sorry I should have written a message to you about this, I totally agree with you [and will definitely do this for the reference implementation that we are hoping to get ready for a near gitcoin hackathon where we want to encourage people to experiment with our harberger tax contracts].

I started working on this; but ran into the issue that these contracts are still upgradable, which makes it more difficult. I agree completely that the erc721 in the future shouldn't be upgradable (juts like the erc20 loyalty token we have added), but we haven't settled 100% how the default transfer function on the erc721 should work (currently it is disabled).

For any new projects the peace mind knowing that you are 100% guaranteed to deploy and setup the corresponding erc721 correctly indeed will be valuable :)